.travis/install.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+
+set -e
+set -x
+
+if [[ "$(uname -s)" == 'Darwin' ]]; then
+    DARWIN=true
+else
+    DARWIN=false
+fi
+
+if [[ "$DARWIN" = true ]]; then
+    brew update
+
+    # xvfb for OS X?
```

Comment:
Please try to only set the xfvb environment variable for linux. We might not need it for OS X. Please test.